internal/common/common.go

```diff
@@ -150,8 +151,19 @@ func GetNetworkCidrAttr(obj interface{}, fieldName string) []*string {
 	}
 
 	funk.ForEach(field, func(elem interface{}) {
-		addresses = append(addresses, swag.String(string(funk.Get(elem, "Cidr").(models.Subnet))))
+		address := funk.Get(elem, "Cidr")
+		if address != nil {
+			addresses = append(addresses, swag.String(string(address.(models.Subnet))))
+		}
 	})
 
 	return addresses
 }
+
+func IsArrayNonEmpty(arg interface{}) bool {
```

Comment:
This is a nit, but I think this should be called `IsSliceNonEmpty()` because it isn't actually an array. That's the reason reflect is required here after playing around with the reflect package a bit today.

Reply:
Totally yes. And given it's a helper specifically for network subnets, I renamed it to be even more specific